scout/dashboard.md

```diff
@@ -29,6 +29,13 @@ Each entry in the list shows the following details:
 - The most recent version of the image and the vulnerabilities for that version. Clicking the link for the base image opens [the image layer view](#image-layer-view).
 - The operating system and architecture of the image.
 - The base image and version used by the repository and the vulnerabilities for that version. Clicking the link for the base image opens [the image layer view](#image-layer-view).
+
+  > **Note**
+  >
+  > Docker Scout detects the base image of an image used in a repository by matching layer `diff_ids`.
```

Comment:
I feel like diff IDs are a bit of an implementation detail - we should obscure that complexity if possible. Maybe just this?

(also suggested removing "in a repository", since this is true of exclusively local images as well)

```suggestion
  > Docker Scout detects the base image of an image by matching layer IDs.
```

Reply:
I like this suggestion. @cdupuis Is this enough clarity/accuracy compromise?

Reply:
Don't think this is technically really correct but it shouldn't matter much. As a reader I wonder what a `layer ID` is. What about just:

```suggestion
  > Docker Scout detects the base image of an image by matching layer content hashes.
```